static/styles/zulip.css

```diff
@@ -42,6 +42,99 @@ a {
     cursor: pointer;
 }
 
+p.n-margin {
+    margin: 10px 0px 0px 0px;
+}
+
+.small-line-height {
+    line-height: 1.1;
+}
+
+.float-left {
```

Comment:
You might want to socialize this technique a bit on the "frontend" stream of Zulip.  I'm not a CSS expert, but I think what you're doing here is a good practice, and we should try to move in this direction.